wiki/Tournaments/COT/2022/ru.md

```diff
@@ -26,16 +26,18 @@ tags:
 | Четвертьфинал | 2022-02-19/2022-02-20 |
 | Полуфинал | 2022-02-26/2022-02-27 |
 | Финал | 2022-03-05/2022-03-06 |
-| Гранд-финал | 2022-03-12/2022-03-13 |
+| Гранд-финал | 2022-03-11/2022-03-12 |
 
 ## Призы
 
-| Место | Приз |
+| Место | Призы |
 | :-: | :-- |
 | ![Gold crown](/wiki/shared/crown-gold.png "Первое место") | 18 месяцев osu!supporter или 3000₽, уникальный бейдж в профиль |
 | ![Silver crown](/wiki/shared/crown-silver.png "Второе место") | 12 месяцев osu!supporter или 2000₽ |
 | ![Bronze crown](/wiki/shared/crown-bronze.png "Третье место") | 4 месяца osu!supporter или 1000₽ |
 
+![](img/badge.jpg "COT 2022 winner badge")
```

Comment:
the title text in quotes should be translated

Reply:
Translated